entities/validation.rb

```diff
@@ -0,0 +1,25 @@
+module Validation
+  def wrong_name?(name)
+    !/^[A-Z]/.match?(name)
```

Comment:
I don't think that is a good idea to call the method `wrong_name?` check for a positive answer and toggle this answer using bang
```
def valid_name?(name)
  /^[A-Z]/.match?(name)
end
```
It looks much better